Description:  manpage-section-mismatch usr/share/man/man1/linphonecsh.1.gz:60 1 != <section
Bug-Debian: http://lintian.debian.org/tags/manpage-section-mismatch.html

Index: linphone-3.5.2/share/C/linphonecsh.1
===================================================================
--- linphone-3.5.2.orig/share/C/linphonecsh.1	2012-02-22 22:54:29.000000000 +1100
+++ linphone-3.5.2/share/C/linphonecsh.1	2012-06-24 22:26:36.000000000 +1000
@@ -57,5 +57,3 @@
 .SH "SEE ALSO"
 .LP 
 linphonec(1) sipomatic(1) linphone(1)
-.TH <program> <section number> "<date>" "" "Linux User's Manual"
-
